### PR TITLE
Release 1.32.0

### PR DESCRIPTION
**Pull request by d4r1091**
# What this PR does

- Adding APMs
  - Interac
  - PayTrail

### CODE REVIEW

File: Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfiguration.swift [hunk 1]
```diff
@@ -23,6 +23,11 @@ class PaymentMethodConfig: Codable {
             .adyenVipps,
             .adyenAlipay,
             .adyenGiropay,
+            .adyenInterac,
+			.adyenPayTrail,
```
Comment (EvansPie) [inline Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfiguration.swift:27]: (⊙.⊙)

File: Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfigurationType.swift [hunk 2]
```diff
@@ -231,6 +243,7 @@ public enum PaymentMethodConfigType: Codable, Equatable {
         case adyenDotPay
         case adyenGiropay
         case adyenIDeal
+        case adyenInterac
```
Comment (EvansPie) [inline Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfigurationType.swift:246]: How come and adyenPayTrail is not needed?

File: Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfiguration.swift (hunk 1 above)
Comment (d4r1091) [reply Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfiguration.swift:27]: you cannot spot that on Xcode but I made a change that will solve it

File: Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfigurationType.swift (hunk 2 above)
Comment (d4r1091) [reply Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfigurationType.swift:246]:
```
        case adyenInterac
        case adyenPayTrail
```

**Review by EvansPie — APPROVED** [review]

**Review by xevious78 — APPROVED** [review]

**Pull request merged**